BinaryObjectScanner/Interfaces/IISOCheck.cs

```diff
@@ -0,0 +1,19 @@
+using SabreTools.Serialization.Wrappers;
+
+namespace BinaryObjectScanner.Interfaces
+{
+    /// <summary>
+    /// Check an ISO for protection
+    /// </summary>
+    public interface IISOCheck<T> where T : WrapperBase
```

Comment:
Somewhat unrelated, but I need to figure out what preferential order an image check would fall compared to executable, content, and path checks. That likely won't be able to be handled in this PR, but if you have suggestions, I'm open to them. The current recommended order is in one of the documentation files.